Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -135,6 +85,84 @@ struct ComposerActionBar: View {
         )
     }
     
+    var defaultMenu: some View {
+        HStack(spacing: 0) {
+            if !showPreview {
+                // Attach Media
+                ImagePickerButton { imageURL in
+                    Task {
+                        await uploadImage(at: imageURL)
+                    }
+                } label: {
+                    Image.attachMediaButton
+                        .foregroundColor(.secondaryTxt)
+                        .frame(minWidth: 44, minHeight: 44)
+                }
+                .padding(.leading, 8)
+                .accessibilityLabel(Text(.localizable.attachMedia))
+                
+                // Expiration Time
+                if let expirationTime, let option = ExpirationTimeOption(rawValue: expirationTime) {
+                    ExpirationTimeButton(
+                        model: option,
+                        showClearButton: true,
+                        isSelected: Binding(get: {
+                            self.expirationTime == option.timeInterval
+                        }, set: {
+                            self.expirationTime = $0 ? option.timeInterval : nil
+                        })
+                    )
+                    .accessibilityLabel(Text(.localizable.expirationDate))
+                    .padding(12)
+                } else {
+                    Button {
+                        subMenu = .expirationDate
+                    } label: {
+                        Image.disappearingMessages
+                            .foregroundColor(.secondaryTxt)
+                            .frame(minWidth: 44, minHeight: 44)
+                    }
+                }
+            }
+
+            Spacer() 
+
+            Text(.localizable.preview)
+                .padding(.horizontal, 10)
+                .foregroundColor(Color.secondaryTxt)
+            Toggle(isOn: $showPreview) {
+                EmptyView()
+            }
+            .labelsHidden()
+        }
+    }
+
+    /// Uploads an image at the given URL to a file storage service.
+    /// - Parameter imageURL: File URL of the image the user wants to upload.
+    private func uploadImage(at imageURL: URL) async {
```

Comment:
Refactored this code in its own function so that we have a smaller `body` and also a less indented uploading code that made it easy to reach the line limit.

Reply:
But the code itself its the same